aider/models/model.py

```diff
@@ -22,7 +22,10 @@ class Model:
     def create(cls, name, client=None):
         from .openai import OpenAIModel
         from .openrouter import OpenRouterModel
+        from .litellm import LiteLLMModel
 
+        if client and not hasattr(client, "base_url"):
```

Comment:
Not sure about this... I setup LiteLLM in docker on a VM for my team to use, the idea being we can setup the models in one place any everyone has access (and all our projects), but this would require setting a URL or I guess a hacky port forward, would be nice to be able to set a URL for litellm in-case its not running locally.

Reply:
This condition doesn't preclude setting a base_url for LiteLLM, but I understand why you'd think that. It's just duck-typing to detect LiteLLM client, which doesn't have a `base_url` property. There's probably a better way (with more clarity) to detect that, so I'll see what I can do.